handbook/marketing/content.md

```diff
@@ -76,6 +76,16 @@ The editorial team will triage pitches weekly on Monday. Once you have submitted
 
 ## Measuring quality 
 
-[Internal feedback form](https://docs.google.com/forms/d/18RhCpBWv6bzQwNEAY8a6BU52aOpCemsodOU-GaGVvmg/edit)
+One of our values is quality and we want to be known as a brand that produces high-quality, interesting content. As a starting point to measure quality, we will use the **[content feedback form](https://docs.google.com/forms/d/18RhCpBWv6bzQwNEAY8a6BU52aOpCemsodOU-GaGVvmg/edit)** which asks our internal subject matter experts and content consumers to rate the quality of the content and how likely they are to share the content. 
+
+For now, we will solicit feedback after the content is published. 
+
+### Using the **[content feedback form](https://docs.google.com/forms/d/18RhCpBWv6bzQwNEAY8a6BU52aOpCemsodOU-GaGVvmg/edit)**
+
+1. Share a link to either the PR or published content in the #progress channel with a link to the feedback form. 
```

Comment:
It says above we will solicit feedback after publishing so for consistency consider removing reference to the PR